rfcs/formats/communication/relay.md

```diff
@@ -12,3 +12,51 @@
 
```

Comment:
Update the `Date` field in every file under `rfcs` that you are making changes in. Use today's date if you wish.